### DISCUSSION TITLE
SignalR C++ Client not being worked on anymore?

### DISCUSSION THREAD

**Discussion by RayKoopa** [dotnet/aspnetcore#58853] [Q&A] [answered]
Is the [SignalR C++ Client](https://github.com/aspnet/SignalR-Client-Cpp/blob/main/README.md#example-usage) still being worked on? Should I be using it in production?

I'm currently looking for alternatives to my current gRPC solution or manual websocket implementations.

I am under the impression the project is dead as there has been very little activity (issues, discussions, commits, tags) in the last 2+ years.

**Comment by davidfowl** [comment, marked as answer]
It's worked on as needed. When there's demand for features then we decide how much to invest.

**Reply by RayKoopa** [reply]
Thank you for the fast reply! I'll report back when I run into an issue.

**Comment by En3Tho** [comment]
I also think it may be worth saying that even if it is not actively developed it is still supported and in scenario where it covers all of your needs but you spot a bug - it will be certainly fixed.
